lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -2179,6 +2179,11 @@ galaxy:
   # <config_dir>.
   #oidc_backends_config_file: oidc_backends_config.xml
 
+  # Sets the prefix for OIDC scopes specific to this Galaxy instance.
+  # If an API call is made against this Galaxy instance using an OIDC bearer token,
+  # it must include a scope with "<prefix>:*". e.g "https://galaxyproject.org/api:*"
+  #oidc_scope_prefix: https://galaxyproject.org/api
```

Comment:
Is the :* missing here?

Reply:
The config setting should specify only the prefix. The `*` is the exact scope that's requested, meaning all permissions. That leaves space for future expansion, like perhaps `<prefix>:datasets` to request access to datasets etc. For now, we only support `<prefix>:*`

Reply:
Should then `e.g "https://galaxyproject.org/api" be changed` to be clear?

Reply:
I've expanded the description a bit. Let me know whether this looks better.

Reply:
Cool, thanks a lot!